src/main/java/teammates/common/datatransfer/FeedbackConstantSumQuestionDetails.java

```diff
@@ -397,16 +398,46 @@ public String getQuestionResultStatisticsHtml(
         Map<String, List<Integer>> optionPoints = generateOptionPointsMapping(responses);
 
         DecimalFormat df = new DecimalFormat("#.##");
+        /*
+        Map<String, List<Integer>> sortedOptionPoints = new TreeMap<String, List<Integer>>(new Comparator<String>() {
+            @Override
+            public int compare(String s1, String s2) {
+                return s2.compareTo(s1);
+            }
+        });
+        */
+        Map<String, List<Integer>> sortedOptionPoints = new TreeMap<String, List<Integer>>();
+        
+        Map<String, String> identifierMap = new HashMap<String, String>();
         
         for (Entry<String, List<Integer>> entry : optionPoints.entrySet()) {
             
+            if (distributeToRecipients) {
+                
+                String participantIdentifier = entry.getKey();
+                String name = bundle.getNameForEmail(participantIdentifier);
+                
+                identifierMap.put(name, participantIdentifier);
+                sortedOptionPoints.put(name, entry.getValue());
```

Comment:
Concatenating the name and email should work with little modifications to the code already done in this PR, so we could go for that.

Another possible solution is to stick to the old code and let the recipient's identifier still be the keys of the `sortedOptionPoints` map. Before iterating over the keys, we could sort the keys with a custom comparator. In the comparator, we compare the names obtained using `bundle.getNameForEmail` (either directly, or stored in another map) instead of the keys directly.